src/ADAL.PCL.iOS/PlatformInformation.cs

```diff
@@ -76,5 +78,20 @@ public override string GetAssemblyFileVersionAttribute()
             // TODO: Check if assembly file version can be read in iOS assembly as well or not. For now, we use assembly version instead.
             return typeof(AdalIdHelper).GetTypeInfo().Assembly.GetName().Version.ToString();
         }
+
+        public override string GetApplicationName()
+        {
+            return NSBundle.MainBundle.InfoDictionary["CFBundleDisplayName"].ToString();
```

Comment:
Returning bundleId instead of DisplayName

---
In reply to: [72857588](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r72857588) [](ancestors = 72857588)